src/blocks/scratch3_looks.js

```diff
@@ -44,7 +45,8 @@ class Scratch3LooksBlocks {
             onSpriteRight: true,
             skinId: null,
             text: '',
-            type: 'say'
+            type: 'say',
+            usageId: null // ID for hiding a timed say/think block.
```

Comment:
Though this comment makes sense in the context of this PR, I don't think it would be clear for someone coming to this without that context. Maybe rephrase to say something along the lines of:
> ID indicating the most recent usage of the say bubble. Used for comparison when determining whether to clear a say/think bubble.